### PR TITLE
feat/pie ios chart

### PR DESCRIPTION
**Pull request by trevorwhealy**
dont forget to npm install to get the new library.

### CODE REVIEW

**Comment by Dane456** [comment]
🐫

File: mobile/Trendwave/app/index.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import React, {Component} from 'react';
 import {View, Text, StyleSheet} from 'react-native';
 
-// Here we can use import statesments using the 
+import PieChart from './components/PieChart.ios.js';
+
```
Comment (danejordan) [inline mobile/Trendwave/app/index.js:5]: should be './components/PieChart'

**Comment by danejordan** [comment]
🚢

**Comment by donovanCarreno** [comment]
:shipit:

**Pull request merged**